package.json

```diff
@@ -7,8 +7,9 @@
   "private": true,
   "scripts": {
     "dev": "next dev",
+    "dev-build": "yarn test:ci && next build",
     "prebuild": "yarn make-tags-sitemap",
-    "build": "yarn test:ci && next build",
+    "build": "yarn dev-build",
```

Comment:
Here I add `dev-build` to do what `build` was doing. Then `build` invokes `yarn dev-build` to get those same steps without duplicating the command. The `pre` and `post` steps, which do the sitemap generation, will still run for `build`, but not for `dev-build`.

Reply:
i think build:dev would be more typical (or dev:build")